server/configuration.js

```diff
@@ -15,17 +15,19 @@ const loginPromise = Promise.resolve(mClient.connect())
     logger.error(`error while logging into mhub (configuration): ${err.message}`)
   })
 
-let clockFormat
+let clockFormat, precount
 
 exports.linkConfiguration = ({ mhub }) => {
   loginPromise
     .then(() => mClient.on('message', message => {
       if (message.topic === CONFIGURATION_TOPIC) {
-        logger.info('Received configurations from mhub')
-        const field = message.data.fields.find(f => f.name === 'clockFormat')
-        if (field) {
-          clockFormat = field.value
-          mhub.sendClockFormat(field.value)
+        logger.info('Received configuration from mhub')
+        const clockFormatField = message.data.clockFormatFields.find(f => f.name === 'clockFormat')
+        const precountField = message.data.clockFormatFields.find(f => f.name === 'precount')
+        if (clockFormatField) {
```

Comment:
Why dont you fall back to default format?

Reply:
This is a mistake

Reply:
So this will be fixed?

Reply:
The fallback happens in the launcher, on which the module must depend. There was another mistake here, which I fixed.